tox.ini

```diff
@@ -1,2 +1,5 @@
 [flake8]
-max-line-length = 100
\ No newline at end of file
+# unfortunately, flake8 does not play with pyproject.tom. what a shame.
+# one day perhaps we'll switch to ruff.
```

Comment:
Make that day today! It's very easy to migrate settings from flake8 to ruff, and also simple to switch out flake8 for ruff in the requirements file.